ooniprobe/Utility/MessageUtility.mm

```diff
@@ -0,0 +1,41 @@
+#import "MessageUtility.h"
+#import "MBProgressHUD.h"
+#import "UIView+Toast.h"
+
+@implementation MessageUtility
+
++ (void)showToast:(NSString*)msg inView:(UIView*)view
+{
+    [view makeToast:NSLocalizedString(msg, nil)];
+}
+
++ (void)alertWithTitle:(NSString *)title message:(NSString *)msg inView:(UIViewController *)view
+{
+    UIAlertController * alert = [UIAlertController
+                                 alertControllerWithTitle:title
+                                 message:msg
+                                 preferredStyle:UIAlertControllerStyleAlert];
+    UIAlertAction* okButton = [UIAlertAction
+                               actionWithTitle:NSLocalizedString(@"ok", nil)
+                               style:UIAlertActionStyleDefault
+                               handler:nil];
+    [alert addAction:okButton];
+    [view presentViewController:alert animated:YES completion:nil];
+}
+
++ (void)alertWithTitle:(NSString *)title message:(NSString *)msg okButton:(UIAlertAction*)okButton inView:(UIViewController *)view
+{
+    UIAlertController * alert = [UIAlertController
+                                 alertControllerWithTitle:title
+                                 message:msg
+                                 preferredStyle:UIAlertControllerStyleAlert];
+    UIAlertAction* cancelButton = [UIAlertAction
+                                   actionWithTitle:NSLocalizedString(@"cancel", nil)
+                                   style:UIAlertActionStyleDefault
+                                   handler:nil];
+    [alert addAction:cancelButton];
+    [alert addAction:okButton];
+    [view presentViewController:alert animated:YES completion:nil];
+}
```

Comment:
Isn't it possible here to refactor code such that one of the two functions calls the other?

Reply:
No because, the okButton has no action in one function, while when there are two buttons the cancel button is the one with no action.
I want to keep as less parameters as possible.

Reply:
ok